src/singUp/components/buton.js

```diff
@@ -0,0 +1,34 @@
+//import libreries
+import React from 'react';
+import {View, StyleSheet, Button} from 'react-native';
```

Comment:
Add spaces at the beginning and at the end of the components

```suggestion
import { View, StyleSheet, Button } from 'react-native';
```